### PR TITLE
Journalist guide revisions

### PR DESCRIPTION
**Pull request by pierwill**
## Status

Ready for review

## Description of Changes

The journalist guide has a todo note: "This whole document needs to be reworked, it's all over the place and does not flow from top to bottom." I've revised and edited the guide fairly extensively.

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/journalist.rst [hunk 1]
```diff
@@ -1,289 +1,295 @@
...
+          and Journalist Workstation <tails_guide>` before continuing.
+
+.. note:: The Tails OS makes using SecureDrop very different from
+          other computing experiences. The added layers of security
+          mean extra steps each time you want to login. With practice,
+          you will become increasingly comfortable with the process.
+	  
+Each journalist has an authenticated Tor hidden service URL for
+logging in to the *Journalist Interface*. This must be done using the
+Tails operating system. After visiting the ".onion" URL and clicking
```
Comment (freddymartinez9) [inline docs/journalist.rst]: `After visiting the ".onion" URL and clicking` This order seems wrong I think. The Journalist should click the Journalist Interface icon on the desktop which will open Tor Browser to the correct .onion page.

File: docs/journalist.rst [hunk 2]
```diff
@@ -1,289 +1,295 @@
...
 
-In order to protect you from malware, the browser only allows you to
-download documents to a special sandbox folder, located at
-``/home/amnesia/Tor Browser``.
+Flagging a Message for Reply
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-|Download to sandbox folder|
+If the server experiences a large number of new sources signing up at
+once and is overloaded with submissions, you will need to flag sources
```
Comment (freddymartinez9) [inline docs/journalist.rst]: Another reason to flag for reply, if a potential source submits something that looks very promising.

File: docs/journalist.rst [hunk 3]
```diff
@@ -1,289 +1,295 @@
...
-copying these documents back to your ``Journalist Workstation``,
-including stripping metadata with MAT.
-
-When you no longer need documents you can right-click on them and choose
-Wipe to delete them.
+If the default application does not work, you can right-click on the
+document and choose **Open with Other Application...** to try opening
+the document with OpenOffice Writer, or Document Viewer. You might
+also need to right-click on a file and choose **Rename...** to rename
+a document with a proper file extension.
```
Comment (freddymartinez9) [inline docs/journalist.rst]: a better line: "for example `.jpg` instead of `.jpeg`"

File: docs/journalist.rst (hunk 1 above)
Comment (pierwill) [reply docs/journalist.rst]: This whole section on flagging for reply seems overly-technical to me. Maybe let's just say that there could be several reasons to flag, e.g. a source looks promising?

File: docs/journalist.rst (hunk 2 above)
Comment (redshiftzero) [reply docs/journalist.rst]: @freddymartinez9 - note what you are describing is not what "flag for reply" does. Flag for reply is used by journalists to indicate which sources should have keys generated if there is a large number of new sources (as the documentation currently describes). This should be left as is. Note there is another feature, the starring of sources which can be used to indicate which sources are of particular interest.

Comment (pierwill) [reply docs/journalist.rst]: This whole section on flagging for reply seems overly-technical to me. Maybe let's just say that there could be several reasons to flag, e.g. a source looks promising?

File: docs/journalist.rst [hunk 4]
```diff
@@ -1,289 +1,299 @@
...
-          could be risky, and what can be done to make this situation
-          better.
+If you do not yet have a GPG key, follow the instructions for your
+operating system to set one up with GnuPG:
 
-If a journalist does not yet have a GPG key, they can follow these
-instructions to set one up with GnuPG (GPG).
+- `GNU/Linux <https://www.gnupg.org/gph/en/manual.html#AEN26>`__
+- `Windows <http://gpg4win.org/>`__
+- `Mac OS X <https://support.gpgtools.org/kb/how-to/first-steps-where-do-i-start-where-do-i-begin>`__
```
Comment (redshiftzero) [inline docs/journalist.rst]: Huh, these are not really guides. We should link to https://ssd.eff.org guides, specifically:

* GNU/Linux: https://ssd.eff.org/en/module/how-use-pgp-linux
* Mac OS: https://ssd.eff.org/en/module/how-use-pgp-mac-os-x
* Windows: https://ssd.eff.org/en/module/how-use-pgp-windows

Note: this is not required for merge, just noting this while I read through this document and noting here for the future.

File: docs/journalist.rst [hunk 5]
```diff
@@ -1,289 +1,299 @@
...
-download documents to a special sandbox folder, located at
-``/home/amnesia/Tor Browser``.
+|Sent reply|
 
-|Download to sandbox folder|
+Once your reply has been successfully submitted, you will be returned
+to the source page and see a message confirming that the reply was
+stored. The source will see your reply the next time they log in with
+their unique codename. To minimize sensitive data retention, the
+source interface UI encourages the source to delete the reply after
```
Comment (redshiftzero) [inline docs/journalist.rst]: Probably better to remove the term "UI"

File: docs/journalist.rst [hunk 6]
```diff
@@ -1,289 +1,299 @@
...
 
-Decrypt and prepare to publish
-------------------------------
+Decrypting and Preparing to Publish
+-----------------------------------
 
-Plug the ``Transfer Device`` into your workstation computer and copy the
-encrypted documents to it. Decrypt them with ``gnupg``.
+Plug the *Transfer Device* into your workstation computer and copy
+over the encrypted documents. Decrypt them with GnuPG.
```
Comment (redshiftzero) [inline docs/journalist.rst]: We've used the term "GPG" throughout, so let's stick with that here instead of "GnuPG"

**Comment by pierwill** [comment]
Thanks @redshiftzero! I've made the requested changes and done a rebase. I'm not sure how to resolve conflicts, though.

**Comment by pierwill** [comment]
I might be able to figure it out, tho. Stand by ;)

File: docs/journalist.rst [hunk 7]
```diff
@@ -1,289 +1,299 @@
...
-          could be risky, and what can be done to make this situation
-          better.
+If you do not yet have a GPG key, follow the instructions for your
+operating system to set one up:
 
-If a journalist does not yet have a GPG key, they can follow these
-instructions to set one up with GnuPG (GPG).
+- `GNU/Linux <https://ssd.eff.org/en/module/how-use-pgp-linux>`__
+- `Windows <https://ssd.eff.org/en/module/how-use-pgp-windows>`__
+- `Mac OS <https://ssd.eff.org/en/module/how-use-pgp-mac-os-x>`__
```
Comment (redshiftzero) [inline docs/journalist.rst]: :metal:

**Comment by redshiftzero** [comment]
Ah... looks like we're having some merge issues. I think you may have either used `git merge` or the "Update PR" button on GitHub instead of `git rebase`. No matter, I'll make a superseding PR to rebase your work on latest `develop` :smile: - note: you will still be listed as the author.

**Comment by pierwill** [comment]
Thanks for clearing it up, @redshiftzero!

**Comment by conorsch** [comment]
Changes merged in #2578. Thanks, @pierwill! The document reads _much_ better now. 😁

**Pull request closed without merging**